### ISSUE TITLE
none

### ISSUE THREAD

**Issue by ghost** [jangsoohyuk/Robust-Federated-Learning-with-Noisy-Labels#2]
Hello, I use the following parameters to train mnist, but the training loss and global model accuracy have not converged. What's the matter?Looking forward to your reply~

('epochs', 1000)
('bs', 64)
('lr', 0.25)
('lr_decay', 0.1)
('schedule', [])
('momentum', 0.5)
('weight_decay', 0.0001)
('feature_dim', 128)
('num_users', 20)
('frac', 0.1)
('local_ep', 5)
('local_bs', 64)
('dataset', 'mnist')
('num_classes', 10)
('iid', False)
('num_shards', 200)
('noise_type', 'symmetric')
('noise_rate', 0.4)
('num_gradual', 10)
('forget_rate', 0.2)
('T_pl', 30)
('lambda_cen', 1.0)
('lambda_e', 0.8)
('gpu', 0)
('seed', 1)
('save_dir', None)
('device', device(type='cuda', index=0))
